wasm/jit/jit_amd64_test.go

```diff
@@ -2636,6 +2639,315 @@ func TestAmd64Compiler_compileMul(t *testing.T) {
 	})
 }
 
+func TestAmd64Compiler_compilClz(t *testing.T) {
+	t.Run("32bit", func(t *testing.T) {
+		for _, tc := range []struct{ in, exp uint32 }{
```

Comment:
I  know you are tired of typing, but maybe dodge "exp" as later my eyes will cross when we have exponents in some instructions..

```suggestion
		for _, tc := range []struct{ input, expected uint32 }{
```

Reply:
or even expectedBitCount as I kindof looked at one of the later functions sideways a second forgetting I am seeing hex on one side and bit count on the other

Reply:
I guess it changes what we are expecting, so maybe expectedUnsetBitCount vs expectedSetBitCount could be the more pedantically ideal name :D

Reply:
https://github.com/tetratelabs/wazero/pull/113/commits/396cd5a5d2313a0f09f1e7c5dea0eba8635cf562 done!